docs/modules/ROOT/pages/python-client-getting-started.adoc

```diff
@@ -0,0 +1,195 @@
+
+cities_map = client.get_map('cities').blocking()
+cities_map.set(1, "London")
+cities_map.set(2, "New York")
+cities_map.set(3, "Tokyo")
+
+entries = cities_map.entry_set()
+
+for key, value in entries:
+    print(f"{key} -> {value}")
+
+client.shutdown()
+----
+
+The output of this snippet is given below:
+
+[source,bash]
+----
+2 -> New York
+1 -> London
+3 -> Tokyo
+----
+
+Following line returns a map proxy object for the `cities` map:
+
+[source, python]
+----
+cities_map = client.get_map('cities').blocking()
+----
+
+If `cities` doesn't exist, it will be automatically created. All the clients connected to the same cluster will have access to the same map.
+
+You may wonder why we have used `blocking()` method over the `get_map()`. This returns a version of this proxy with only blocking
+(sync) method calls, which is better for getting started. For async calls, please check our https://hazelcast.readthedocs.io/en/stable/#usage[API documentation].
+
+With these lines, client adds data to `cities` map. The first parameter is the key of the entry, the second one is the value as mentioned above.
+
+[source, python]
+----
+cities_map.set(1, "London")
+cities_map.set(2, "New York")
+cities_map.set(3, "Tokyo")
+----
+
+Then, we get the values with the `entry_set()` method and iterate over the set.
+
+[source, python]
+----
+entries = cities_map.entry_set()
+
+for key, value in entries:
+    print(f"{key} -> {value}")
+----
+
+Finally, `client.shutdown()` terminates our client and release its resources.
+
+== Add a Listener to the Map
+
+You can add an entry listener using the `add_entry_listener()` method available on the map proxy object.
+This will allow you to listen to certain events that happen in the map across the cluster.
+
+The first argument to the `add_entry_listener()` method is `includeValue`.
+This boolean parameter, if set to true, ensures the entry event contains the entry value.
+
+The second argument to the `add_entry_listener()` method is an object that is used to define listeners.
+In this example, we register listeners for the `added`, `removed` and `updated` events.
+
+This enables your code to listen to map events of that particular map.
+
+[source, python]
+----
+import hazelcast
+
+def entry_added(event):
+    print(f"Entry added with key: {event.key}, value: {event.value}")
+
+
+def entry_removed(event):
+    print(f"Entry removed with key: {event.key}")
+
+
+def entry_updated(event):
+    print(f"Entry updated with key: {event.key}, old value: {event.old_value}, new value: {event.value}")
+
+client = hazelcast.HazelcastClient()
+
+cities_map = client.get_map('cities').blocking()
+cities_map.add_entry_listener(
+    include_value=True, added_func=entry_added, removed_func=entry_removed, updated_func=entry_updated
+)
+
+cities_map.clear()
+cities_map.set(1, "London")
+cities_map.set(2, "New York")
+cities_map.set(3, "Tokyo")
+
+cities_map.remove(1)
+cities_map.replace(2, "Paris")
+
+cities_map.(1)
```

Comment:
Did you mean `cities_map.get(1)`

Reply:
All our guides have the version specified - this is a consistency copy from C# and Java.

Suggest fixing them ALL after the first cut is made available.